app/views/transition_landing_page/_take_action.html.erb

```diff
@@ -25,19 +25,19 @@
       <div class="govuk-grid-column-one-third">
         <ul class="govuk-list landing-page__take-action-traffic-lights">
           <li>
-            <%= image_tag 'take-action-red.svg', class: "landing-page__take-action-traffic-lists-icon" %>
+            <%= image_tag 'take-action-red.svg', class: "landing-page__take-action-traffic-lists-icon", alt: "" %>
```

Comment:
Why `alt: ""` ?

Reply:
I saw @DilwoarH [asked about this in frontenders last week](https://gds.slack.com/archives/CARGH33JS/p1594629065068800)

I think @injms said it stops screen readers reading out the source for the image. But as it has no semantic information there's nothing to put in there that would be otherwise useful for a screen reader to read out.

Reply:
Without the `alt` attribute some screenreader read out the image `src` instead - a blank `alt` stops this.